### ISSUE TITLE
Virtua Fighter 5 (I don't know the game id or what version im using)

### ISSUE THREAD

**Issue by jonrod2020** [xenia-project/game-compatibility#93]
http://marketplace.xbox.com/en-US/Product/Virtua-Fighter-5/66acd000-77fe-1000-9115-d802534507ee

state: menu

I dont know which version of cydia im using

It's very very buggy. 

Issues:
General graphical issues.
Crashes when pressing start.

![image of what the menu looks like](http://i.snag.gy/1QkS4.jpg)

**Comment by voltagex** [comment]
PAL or NTSC?

**Comment by voltagex** [comment]
Hey @Razzile, I don't know if this is the correct ID. The copy of VF5 I've borrowed has that ID but crashes before the menu in the screenshot.

**Comment by Razzile** [comment]
different versions of xenia will have different results. By the looks of all your reports it would seem your xenia build is unstable as everything is either state-nothing or state-load :P

**Comment by LarsMasters** [comment]
Means Final Showdown can't too

**Comment by Sleepy93** [comment]
# Update
The log was a bit old so I decided to upload a new one.

# Tested
On (https://github.com/benvanik/xenia/commit/a148b965f19a99e4ea6c346075bd19de19f085fc)

# Log
File(http://www.mediafire.com/file/nz38mznaaae6qaq/VirtuaFighter5_161030.log)

# Issue 
 crash after start with Xenia Stopped working

# Labels
- regression
- crash

**Comment by voltagex** [comment]
Please, please don't use unreliable filehosters for logs. You can now attach files directly to the issue, like this - 
[VirtuaFighter5_161030.zip](https://github.com/xenia-project/game-compatibility/files/560875/VirtuaFighter5_161030.zip)

**Comment by Sleepy93** [comment]
@voltagex okay sorry for it :)

**Comment by Sleepy93** [comment]
# Update
crash with Xenia stopped working.
>Unhandled exception at 0x000000006DFBF0F0 (nvoglv64.dll) in xenia.exe: Fatal program exit requested.

# Tested 
On (https://github.com/benvanik/xenia/commit/70bd283e80125932339f23a54e5efcb3c74f0456)

# Log file
[VirtuaFighter5_70bd283e_161210.txt](https://github.com/xenia-project/game-compatibility/files/644051/VirtuaFighter5_70bd283e_161210.txt)

# Labels
(still)
- regression
- crash

**Comment by Sleepy93** [comment]
# Update
game starts goes into menu. 
There are a lot of gpu corruptions, and missing drawings so can't see ingame... 

![virtua](https://user-images.githubusercontent.com/20132031/34909495-8367b8b4-f8a2-11e7-8cd4-e0b6c2b41162.png)

# Tested 
On (https://github.com/benvanik/xenia/commit/d512d4127fa12a710e5d1d6d8067fe050b3445e7)

# Issues

> !> 00000028 ResolvePath(ShaderDumpxe:\CompareBackEnds) failed - no root found
> !> 00000004 PM4_DRAW_INDX_IMM(1, 1): Failed in backend

# GPU

w> 00000004 GPU: Write to unknown register (0081 = 80010000)
w> 00000004 GPU: Write to unknown register (0082 = 00000000)
w> 00000004 GPU: Write to unknown register (1E4E = 00000000)

# Log file
[virtua.txt](https://github.com/xenia-project/game-compatibility/files/1628696/virtua.txt)

# Labels
- state-gameplay
- gpu-missing-drawing
- gpu-corrupt-drawing

**Comment by Blayer98** [comment]
Weird, I'm still having this exact same issue, even on the latest version, which is https://github.com/benvanik/xenia/commit/5477b1e40fcbce42024e6080d87917df60c8ec48

**Comment by Junior5a** [comment]
Updated on https://github.com/xenia-project/xenia/commit/aa73d652ba535eee112bb9284e651accdc3481c3

Shown black screen when playing the game.
![image](https://user-images.githubusercontent.com/31457368/82814468-2666ba80-9e8f-11ea-9b72-22f5a3cf553f.png)

**Comment by videoware2000** [comment]
>  
> Shown black screen when playing the game.
Yeah, worked fine in Dec 4 2019 release, only had an audio issue where game could go silent any moment, with comeback or not, pretty good performance only a bit laggy sometimes.

However in the latest release game stopped showing 3d graphics and sound effects are missing completely, F5 doesn't solve.

Regards.

PS: I add screen from version 4-12-2019.
![234234234](https://user-images.githubusercontent.com/66091981/83193656-b6c72880-a137-11ea-87a8-2ac715b0f3d1.png)

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/550
